tests/pybaseball/conftest.py

```diff
@@ -21,3 +22,19 @@ def get_dataframe(filename: str) -> pd.DataFrame:
         return pd.read_csv(os.path.join(data_dir, filename), index_col=0).reset_index(drop=True)
 
     return get_dataframe
+
+@pytest.fixture()
+def response_get_monkeypatch(monkeypatch):
```

Comment:
Actually, just the one thing: can you please type annotate this and add a docstring for this function?

Reply:
You read my mind (or I read your mind?). Just added the annotations. I'll add the docstring.